src/pages/academy/index.tsx

```diff
@@ -94,22 +94,37 @@ const DerivBlog = ({ data }: DerivBlogProps) => {
     let non_featured_video_list_data = data.directus.videos
     let featured_video_list_data = data.directus.featured_video
 
+    const {
+        market_news_eu,
+        recent_eu,
+        featured_eu,
+        homepage_banners_eu,
+        videos_eu,
+        featured_video_eu,
+        market_news_uk,
+        recent_uk,
+        featured_uk,
+        homepage_banners_uk,
+        videos_uk,
+        featured_video_uk,
+    } = data.directus
+
     // We need to include the !is_uk_country check together with is_eu_country because 'gb'
     // is a valid country code for both EU and UK in our country base.
```

Comment:
Let's update this comment. This doesn't makes sense anymore since we are not using the old country checker and I suppose we can just omit it.

Reply:
Done